testutils/clustermanager/gke_test.go

```diff
@@ -119,7 +119,7 @@ func (fgsc *FakeGKESDKClient) create(project, location string, rb *container.Cre
 	return fgsc.newOp(), nil
 }
 
```

Comment:
Can we test zonal cluster operations too?

Reply:
Since this is all faked, there is not really a good way of testing zonal cluster